internal/service/resourcepolicy/data_source_schema.go

```diff
@@ -0,0 +1,89 @@
+package resourcepolicy
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: dataSourceSchema(false),
+	}
+}
+
+func dataSourceSchema(isPlural bool) map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"created_by_user": schema.SingleNestedAttribute{
+			Description: "The user that last updated the atlas resource policy.",
+			Computed:    true,
+			Attributes: map[string]schema.Attribute{
+				"id": schema.StringAttribute{
```

Comment:
aren't we defining MarkdownDescription as well?

Reply:
Added in [e1e988f](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2563/commits/e1e988fc2a97f90ef85314871ae78d3f1ffaf622).
Hopefully, one day we only need one `Description` 😅